### PR TITLE
Replace some instances of formatWithUnit with formatHtmlWithUnit

### PR DESCRIPTION
**Pull request by fanquake**
Strings in a HTML context should be using formatHtmlWithUnit.

### CODE REVIEW

**Comment by dcousens** [comment]
light utACK

File: src/qt/coincontroldialog.cpp [hunk 1]
```diff
@@ -637,14 +637,14 @@ void CoinControlDialog::updateLabels(WalletModel *model, QDialog* dialog)
...
+    toolTip1 += tr("This means a fee of at least %1 per kB is required.").arg(BitcoinUnits::formatHtmlWithUnit(nDisplayUnit, CWallet::GetRequiredFee(1000))) + "<br /><br />";
     toolTip1 += tr("Can vary +/- 1 byte per input.");
 
     QString toolTip2 = tr("Transactions with higher priority are more likely to get included into a block.") + "<br /><br />";
     toolTip2 += tr("This label turns red if the priority is smaller than \"medium\".") + "<br /><br />";
-    toolTip2 += tr("This means a fee of at least %1 per kB is required.").arg(BitcoinUnits::formatWithUnit(nDisplayUnit, CWallet::GetRequiredFee(1000)));
+    toolTip2 += tr("This means a fee of at least %1 per kB is required.").arg(BitcoinUnits::formatHtmlWithUnit(nDisplayUnit, CWallet::GetRequiredFee(1000)));
 
-    QString toolTip3 = tr("This label turns red if any recipient receives an amount smaller than %1.").arg(BitcoinUnits::formatWithUnit(nDisplayUnit, ::minRelayTxFee.GetFee(546)));
+    QString toolTip3 = tr("This label turns red if any recipient receives an amount smaller than %1.").arg(BitcoinUnits::formatHtmlWithUnit(nDisplayUnit, ::minRelayTxFee.GetFee(546)));
```
Comment (jonasschnelli) [inline src/qt/coincontroldialog.cpp]: Not related to this PR: but we should remove the hardcoded `546` here. I think just removing the number and changing the text to something like "This label turns red if any recipient receives an amount smaller then the **current dust threshold**") .

Comment (jonasschnelli) [inline src/qt/coincontroldialog.cpp]: @fanquake: mind packing this into this PR?

Comment (MarcoFalke) [inline src/qt/coincontroldialog.cpp]: Looks like it's already used in this cpp file. So could as well make it calculate the threshold every time? https://github.com/fanquake/bitcoin/blob/5fdf32de7ed85a1a0aec7cdedb83f750f4a0f7ff/src/qt/coincontroldialog.cpp#L577

**Comment by paveljanik** [comment]
utACK

**Comment by jonasschnelli** [comment]
ACK 6fd0a079d8f9fa552d84be7a6787b1165b99b302

**Pull request merged**